test/e2e/terminal.test.ts

```diff
@@ -0,0 +1,77 @@
+import { test, expect } from "@playwright/test"
+import * as fs from "fs"
+import { tmpdir } from "os"
+import * as path from "path"
+import util from "util"
+import * as cp from "child_process"
+import { STORAGE } from "../utils/constants"
+import { CodeServer } from "./models/CodeServer"
+
+test.describe("Integrated Terminal", () => {
+  // Create a new context with the saved storage state
+  // so we don't have to logged in
+  const options: any = {}
+  const testFileName = "test.txt"
```

Comment:
Inconsequential, but this matches a bit better with the switch to the pipe method:

```suggestion
  const testFileName = "pipe"
```